Package.swift

```diff
@@ -8,10 +8,10 @@ let depNimble: PackageDescription.Target.Dependency = .product(name: "Nimble", p
 let package = Package(
     name: "Mobius",
     platforms: [
-        .iOS(.v10),
-        .macOS(.v10_12),
-        .tvOS(.v10),
-        .watchOS(.v3),
+        .iOS(.v13),
+        .macOS(.v10_15),
+        .tvOS(.v13),
+        .watchOS(.v6),
```

Comment:
Nimble complained about some of these. Probably about time to bump anyway? iOS 10 is mighty old.